### PR TITLE
Removing duplicate sections

### LINKED ISSUE #2308: Remove duplicate section
This [topic](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/general/angular-grid-overview-guide.html) has 2 sections, that are duplicate with 2 sections in this [one](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grids_and_lists.html)

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #2308

### CODE REVIEW

**Review by zdrawku — COMMENTED** [review]
@aatanasov2 do you think that we should remove entirely the two paragraphs, after all the topic was meant to give an overview of the grid and its usage and we are going to remove that opening just like that?

Should we consider adding something else instead, like a brief description of the topic or a reference to the Grids and lists topic? For example:

_"Get to know the Angular Data Grid and how to use it [by checking out this informative section](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grids_and_lists.html#what-is-an-angular-data-grid) part of our Grid Overview topic.."_ .. or something like that :)

**Comment by aatanasov2** [comment]
@zdrawku It makes sense to keep the reference to the data grid page.
 I see this page as a natural flow to all angular components with some generic content around the angular as a whole:
-  what is angular
- what is the latest version of angular
- how to start with angular app development
- What is Ignite UI for Angular
- What you can do with Ignite UI ( links to grids, charts, and other key features)

File: en/components/general/angular-grid-overview-guide.md [hunk 1]
```diff
@@ -6,16 +6,6 @@ _keywords: angular, angular app development, infragistics
 
 # A Complete Guide to Angular Grid and Angular App Development
```
Comment (zdrawku) [inline en/components/general/angular-grid-overview-guide.md:8]: Please add here the following sentance - _Get to know the Angular Data Grid and how to use it [by checking out this informative section](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grids_and_lists.html#what-is-an-angular-data-grid) part of our Grid Overview topic._

**Review by zdrawku — CHANGES_REQUESTED** [review]

File: en/components/general/angular-grid-overview-guide.md (hunk 1 above)
Comment (HristoP96) [reply en/components/general/angular-grid-overview-guide.md:8]: > Please add here the following sentance - _Get to know the Angular Data Grid and how to use it [by checking out this informative section](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grids_and_lists.html#what-is-an-angular-data-grid) part of our Grid Overview topic._

Applied :)

**Comment by zdrawku** [comment]
Aleks has some more recommendations for this topic, we should discuss them and apply them later on.

**Review by zdrawku — APPROVED** [review]

**Pull request merged**